lib/Content/View/ContentCreateSuccessView.php

```diff
@@ -25,4 +30,20 @@ public function __construct(Response $response)
         $this->setResponse($response);
         $this->setControllerReference(new ControllerReference('ez_content_edit:createWithoutDraftSuccessAction'));
     }
+
+    /**
+     * @param \eZ\Publish\API\Repository\Values\Content\Location|null $location
+     */
+    public function setLocation(?Location $location)
```

Comment:
Not sure if there is any case when this is set to null

Reply:
Yep, just wanted to be consistent with https://github.com/ezsystems/repository-forms/blob/master/lib/Content/View/ContentEditView.php#L61, but it seems that indeed there is no case for that. Let's wait for @webhdx 🙂

Reply:
Only hypothetical case would be when you want to remove location.

For getLocation it's needed for cases where there are no locations _(most notably draft, but also technically possible with location less content)_, but I also don't see a strong need to support this on setLocation().